flexmeasures/api/v3_0/tests/test_sensor_schedules.py

```diff
@@ -327,3 +327,142 @@ def test_trigger_and_get_schedule(
 
     # Check whether the soc-at-start was persisted as an asset attribute
     assert sensor.generic_asset.get_attribute("soc_in_mwh") == start_soc
+
+
+@pytest.mark.parametrize(
+    "target_soc, charging_station_name",
+    [
+        (9, "Test charging station"),
+        (15, "Test charging station"),
+        (5, "Test charging station (bidirectional)"),
+        (15, "Test charging station (bidirectional)"),
+    ],
+)
+def test_get_schedule_fallback(
+    app,
+    target_soc,
+    charging_station_name,
+    add_market_prices,
+    add_battery_assets,
+    battery_soc_sensor,
+    add_charging_station_assets,
+    keep_scheduling_queue_empty,
+):
+    """
+    Test if the fallback job is created after a failing StorageScheduler call. This test
+    is based on flexmeasures/data/models/planning/tests/test_solver.py
+    """
+
+    start = "2015-01-02T00:00:00+01:00"
+    epex_da = Sensor.query.filter(Sensor.name == "epex_da").one_or_none()
+    charging_station = add_charging_station_assets[charging_station_name].sensors[0]
+
+    assert charging_station.get_attribute("capacity_in_mw") == 2
+    assert charging_station.get_attribute("market_id") == epex_da.id
+
+    # create a scenario that yields an unfeasible problem
```

Comment:
infeasible

Reply:
Can we quickly mention what made the problem infeasible (but still pass validation)?